float64[] s
float64[] n
float64[] mu
float64[] vx_c
float64[] vy_c
float64[] dpsi
float64[] del_s
float64[] fx_m
float64[] blending_factor